### ISSUE TITLE
PickerCell showing label of different PickerCell

### ISSUE THREAD

**Issue by TrueGeek** [muak/AiForms.SettingsView#166]
### Description

We have a page with a single SettingsView that has a large number of Sections. We've only noticed this problem on this page, and only on Android. If you scroll all the way down and then all the way back up often one or more of the PickerCells will display the text of a different PickerCell. The internal value says the same, luckily, it's just the label that's wrong.

### Steps to Reproduce

1. Create a page with a SettingsView
2. Add a ton of sections
3. In each section put a PickerCell with different collections
4. Run on Android and scroll to the bottom, then the top

### Platforms

- [x ] Android
- [ ] iOS

### Basic Information

- AiForms.SettingsView 1.3.21
- Xamarin.Forms 5.0.0.2012
- Affected Devices: Android only

**Comment by ivoryguard** [comment]
I also experienced the issue with AiForms.SettingsView 1.3.18 or higher.

I found a workaround.

It may occur when a PickerCell is added to the last Section and other PickerCells are added to above Sections.
When I moved the PickerCell in the last Section to first Section, the issue disappeared.

**Comment by ivoryguard** [comment]
> I also experienced the issue with AiForms.SettingsView 1.3.18 or higher.
> 
> I found a workaround.
> 
> It may occur when a PickerCell is added to the last Section and other PickerCells are added to above Sections.
> When I moved the PickerCell in the last Section to first Section, the issue disappeared.

My workaround does not work actually. The issue may occur arbitrarily according to the order and count of cells.

**Comment by ivoryguard** [comment]
It may be related to the virtualization on scrolling.
I splitted some settings to new sub pages in order to decrease content height and the issue was gone.

**Comment by muak** [comment]
@TrueGeek 
Thank you for your report.

How number of sections does occur that issue?

I coudn't reproduce it using 20 sections.

**Comment by AlvinKwong** [comment]
I also experienced this issue with using the latest AIForms.SettingView 1.3.27.

I have 7 sections of it but I don't think it relates to how many sections the view has.
It happens that when the page is long enough that allows to scroll down and some picker cells are out of the screen. And when scrolls up back, those picker cells will display incorrect texts. (not the title but the selected item)

I was testing it on a Samsung S9+ which is Android 10.
Attached is a video I took on the screen when this happens while scrolling. Hope it helps to identify the issue.

https://user-images.githubusercontent.com/31871670/132653112-7f6c5416-a1d2-4537-9ea3-b3f604cbaf4f.mp4
